packages/core/strapi/src/commands/utils/logger.ts

```diff
@@ -107,6 +124,34 @@ const createLogger = (options: LoggerOptions = {}): Logger => {
 
       return ora(text);
     },
+
+    progressBar(totalSize: number, text: string) {
+      if (silent) {
+        return {
```

Comment:
you could put that in a const outside the function call

Reply:
I think this was included here because of consistency with the spinner and because it relies on the `silent` arg present in the scope. What would be the advantage of extracting it from the fn call?

Reply:
everytime you call the progressBar (or any of the other function where I commented) you recreate the object, just not efficient even if impact should be pretty low